src/server/models/match.model.js

```diff
@@ -0,0 +1,28 @@
+import mongoose from 'mongoose';
+const Schema = mongoose.Schema;
+
+let matchSchema = new Schema({
+    mentorId: Schema.Types.ObjectId,
```

Comment:
can we make the type here to be userSchema.userId? Then if we decide to change the format of userId, we can just change userSchema only.